### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by sudeepman**

### CODE REVIEW

File: ankita.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This is our Git Session
```
Comment (sudeepman) [inline ankita.txt:1]: this is ok

Comment (bikudagr8) [inline ankita.txt:1]: I dont understand this!!!

Comment (pshrestha) [inline ankita.txt:1]: this looks good..

File: pratik.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+This is Pratik Shrestha.
+
+This is git practice.
+
+This is new edit for dev branch.
```
Comment (sudeepman) [inline pratik.txt:5]: good to go
done

File: ankita.txt (hunk 1 above)
Comment (Ankita00) [inline ankita.txt:1]: yes

Comment (eugeneshrestha) [inline ankita.txt:1]: I don't understand this either!

**Pull request merged**